src/views/bootablevolumes/list/BootableVolumesList.tsx

```diff
@@ -44,6 +46,15 @@ const BootableVolumesList: FC = () => {
     preferences,
   );
 
+  // This is to select the 'Hide deprecated bootable volumes' filter by default
+  useEffect(() => {
+    onFilterChange('hideDeprecatedBootableVolumes', {
```

Comment:
please put hideDeprecatedBootableVolumes in a const as it repeats multiple times

Reply:
best to have all hard-coded strings should be in consts

Reply:
Fixed!